package.json

```diff
@@ -36,53 +36,53 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@angular/animations": "7.2.15",
-    "@angular/common": "7.2.15",
-    "@angular/compiler": "7.2.15",
-    "@angular/compiler-cli": "7.2.15",
-    "@angular/core": "7.2.15",
-    "@angular/forms": "7.2.15",
-    "@angular/http": "7.2.15",
-    "@angular/platform-browser": "7.2.15",
-    "@angular/platform-browser-dynamic": "7.2.15",
-    "@angular/router": "7.2.15",
-    "@blackbaud/auth-client": "2.17.2",
-    "@pact-foundation/pact": "9.2.0",
+    "@angular/animations": "7.2.16",
+    "@angular/common": "7.2.16",
+    "@angular/compiler": "7.2.16",
+    "@angular/compiler-cli": "7.2.16",
+    "@angular/core": "7.2.16",
+    "@angular/forms": "7.2.16",
+    "@angular/http": "7.2.16",
+    "@angular/platform-browser": "7.2.16",
+    "@angular/platform-browser-dynamic": "7.2.16",
+    "@angular/router": "7.2.16",
+    "@blackbaud/auth-client": "2.22.0",
+    "@pact-foundation/pact": "9.6.1",
     "@pact-foundation/pact-web": "7.4.0",
-    "@skyux-sdk/builder": "3.10.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0",
+    "@skyux-sdk/builder": "3.14.0",
```

Comment:
Mind bumping this to `3.15.0`. It includes an important feature for libraries.

Reply:
Done.